pshtt/pshtt.py

```diff
@@ -16,6 +16,7 @@
 import pytablewriter
 import sys
 import codecs
+import OpenSSL
```

Comment:
I installed `pyopenssl` 17.2.0 and froze it to requirements.txt, and this fixed it.